bridges/TheWhiteboardBridge.php

```diff
@@ -0,0 +1,21 @@
+<?php
+class TheWhiteboardBridge extends BridgeAbstract {
+	const NAME = 'The Whiteboard';
+	const URI = 'https://www.the-whiteboard.com/';
+	const DESCRIPTION = 'Get the latest comic from The Whiteboard';
+	const MAINTAINER = 'CyberJacob';
+
+	public function collectData() {
+		$item = array();
+
+		$html = getSimpleHTMLDOM(self::URI) or returnServerError('Could not load The Whiteboard.');
+
+		$image = $html->find('center', 1)->find('img', 0);
+		$image->src = self::URI . '/' . $image->src;
+
+		$item['title'] = explode("\r\n", $html->find('center', 1)->plaintext)[0];//->find('text')->plaintext;
+		$item['content'] = $image;
+
```

Comment:
You should set a timestamp for each item.
`$item['timestamp'] = explode("\r\n", $html->find('center', 1)->plaintext)[0];`

Reply:
That's a good suggestion, thanks. I've added it to the pull request.